README.md

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 <p align="center">
     <br>
     <a href="https://layer.ai">
-        <img src="https://app.layer.ai/assets/logo.svg" width="200" alt="Layer"/>
+          <img src=".github/assets/layer_logo_light.png" width="512" alt="Layer"/>
```

Comment:
This readme is uploaded to https://pypi.org/project/layer/ so these images won't load there if they are loaded from local files.